cli/pcluster/config_sanity.py

```diff
@@ -49,26 +49,23 @@ def __get_partition(self):
     @staticmethod
     def __check_sg_rules_for_port(rule, port_to_check):
         """
-        Verify if the security group rule accepts connections to the given port.
+        Verify if the security group rule accepts connections on the given port.
 
         :param rule: The rule to check
         :param port_to_check: The port to check
         :return: True if the rule accepts connection, False otherwise
         """
-        port = rule.get("FromPort")
-        ip_rules = rule.get("IpRanges")
-        group = rule.get("UserIdGroupPairs")
-
-        is_valid = False
-        for ip_rule in ip_rules:
-            ip = ip_rule.get("CidrIp")
-            # An existing rule is valid for EFS if, it allows all traffic(0.0.0.0/0)
-            # from all ports or the given port, and does not have a security group restriction
-            if (not port or port == port_to_check) and ip == "0.0.0.0/0" and not group:
-                is_valid = True
-                break
-
-        return is_valid
+        from_port = rule.get("FromPort")
+        to_port = rule.get("FromPort")
```

Comment:
you are assigning`get("FromPort")` for both "from_port" and "to_port" variables.

Reply:
fixed